README.rst

```diff
@@ -96,14 +96,17 @@ Quick Example
     from nextcord.ext import commands
 
 
-    bot = commands.Bot(command_prefix='$')
+    bot = commands.Bot()
 
-    @bot.command()
-    async def ping(ctx):
-        await ctx.reply('Pong!')
 
-    bot.run('token')
+    @bot.slash_command(description="Replies with pong!")
+    async def ping(interaction: nextcord.Interaction):
+        await interaction.send("Pong!", ephemeral=True)
 
+    bot.run("token")
+
+
+It is recommended that you use ``commands.Bot`` if you wish to use prefix commands, as although both ``nextcord.Client`` and ``commands.Bot`` allow you to interface with Discord, ``commands.Bot`` provides an easier way to make prefix commands with a framework. Make sure to not use both versions, though, as only one of them will actually be connected to Discord.
```

Comment:
I'm not too sure how I feel about this line overall. It just throws information onto someone who has only just seen the library (otherwise why would they read the example) which they do not need from the README. The docs can be used instead to demonstrate this.

Reply:
I agree with this, just having a simple example is enough. (It also can be confusing since it says commands.Bot is recommended for prefix commands, but the example uses commands.Bot for slash. This is fine, but can be confusing to read them together.)

Reply:
> I agree with this, just having a simple example is enough. (It also can be confusing since it says commands.Bot is recommended for prefix commands, but the example uses commands.Bot for slash. This is fine, but can be confusing to read them together.)

I've updated the file.

Reply:
Now it is just downright incorrect, `commands.Bot` provides no extra slash-specific additions.

Reply:
What do you mean? bot.slash_command uses commands.Bot

Reply:
`Client.slash_command` also exists, your description on why `commands.Bot` is good is just unnecessarily complex. There is no need for it, users haven't even used the library yet if they are reading the example.

Reply:
I think it's probably best to just remove the paragraph and maybe add something in the main documentation in another PR

Reply:
> I think it's probably best to just remove the paragraph and maybe add something in the main documentation in another PR

Did that just now.